dist/components/import.html

```diff
@@ -0,0 +1,2 @@
+<link rel="import" href="/vendor/polymer/polymer.html">
+<link rel="import" href="/components/top-bar/top-bar.html">
```

Comment:
Actually, this file is not meant to be used in this way. It's solely for creating the compiled import-built file. I should be clear about that in a comment.